questions/qa-when-lang-neg.en.html

```diff
@@ -101,6 +101,12 @@ <h3>Shortcomings of language negotiation</h3>
       sub-sections. Note however that some measure of cultural adaptation (e.g. changing the currency) does not necessarily make pages non-equivalent; the
       non-equivalence limitation to language negotiation really exists when a <em>site</em> is adapted so that pages in different languages do not
       correspond to one another any more.</p>
+
+    <section id="searchengines">
+    <h4>Search engine discovery challenges</h4>
```

Comment:
This section makes a number of assumptions about how language negotiated pages are structured, stored, and served. Those assumptions may not hold true in practice and are an important consideration in how a given site structures its language negotiation. For example, in my proposed article about language negotiation, I call out the need for domain, path, or query based language identification. One reason for this is to expose per-page language/locale to robots and crawlers.

This also helps with things like offline links in a specific language (if I have a display ad in Spanish, I want the link to to go the Spanish language experience page)